Softeq.XToolkit.Permissions.Droid/PermissionsManager.cs

```diff
@@ -43,19 +43,19 @@ public void SetPermissionDialogService(IPermissionsDialogService permissionsDial
                                         ?? throw new ArgumentNullException(nameof(permissionsDialogService));
         }
 
-        private bool IsPermissionRequested<T>()
+        private bool IsPermissionDeniedEver<T>()
             where T : BasePermission
         {
-            return Preferences.Get(GetPermissionRequestedKey<T>(), false);
+            return Preferences.Get(GetPermissionDeniedEverKey<T>(), false);
         }
 
-        private void SetPermissionRequested<T>(bool value)
+        private void SetPermissionDenied<T>(bool value)
             where T : BasePermission
         {
-            Preferences.Set(GetPermissionRequestedKey<T>(), value);
+            Preferences.Set(GetPermissionDeniedEverKey<T>(), value);
         }
 
-        private string GetPermissionRequestedKey<T>()
+        private string GetPermissionDeniedEverKey<T>()
             where T : BasePermission
         {
             return $"{nameof(PermissionsManager)}_IsPermissionRequested_{typeof(T).Name}";

```

Comment:
Should we also update key value? Now that the meaning has changed - previous values are not valid either way

Reply:
May have problems with backward compatibility. If user already press "don's show never again", system permission will never shown, and we can not navigate user to settings

Reply:
Migration added